docs/build/tinlake.md

```diff
@@ -0,0 +1,37 @@
+---
+id: tinlake
+order: 2
+title: Tinlake
+---
+
+> The documentation of Tinlake’s smart contract is currently being updated.
```

Comment:
@denniswell ?

Reply:
@lucasvo Because the old documentation is v2. This is not about "not up to date" but a very different version. If you still want to add it. I'd ship as it is and ask Sounak to add afterwards

Reply:
I don't think the version is too different to be honest. The fundamentals are still the same. But I guess we are out of time now. The problem is we now have nothing about the smart contracts themselves.

Reply:
Can we leave the old documentation up and link to it for now?

Reply:
no we can't

Reply:
technically not possible.